### DISCUSSION TITLE
hacked my account

### DISCUSSION THREAD

**Discussion by Lordkyllua** [community/community#38140] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi everyone. A few days ago my other account here was hacked.
How can i get it back.
github not responding. and I have proof.
![Screenshot_20221031-162343_2](https://user-images.githubusercontent.com/117127321/199685364-82a6976f-debc-430e-adc5-c9b9924f8e5f.png)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Lordkyllua 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
